_summaries/touvron2020training_1.md

```diff
@@ -1,19 +1,47 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: touvron2020training_1
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea?  <br/>
+Vision Transformer can achieved excellent results on image classification, but it requires large training datasets and computationally expensive. The authors of this paper came up with a data-efficient image transformer (DeiT) that use ImageNet as training datasets, which has same architecture as Vision Transformer, but by using different training methods and distillation, DeiT has better performance even than EfficientNet (see the following graph). In addition, DeiT is significantly less computationally expensive than Vision Transformer as well. 
+
+<img width = 600 alt = "training_Deit_1" src = "training_Deit_1.png">
+
+* How is it realized (technically)? <br/>
+First, the distillation method requires one image classification model to be a teacher, and simply add one distillation token so that it can interact with class token and patch tokens in self-attention layers. As shown in the following graph <br/>
+<image width = 300 alt = "training_Deit_2" src = "training_Deit_2.png">
+
+There are actually two type of distillation methods -- soft distillation and hard distillation. The main difference between them is they use different objective functions.
```

Comment:
Equations are hard to read, so include a high level interpretation. One such interpretation could be that soft means the student should try to mimic all of the teacher's probabilities, including those of the wrong classes, while hard means the student should only try to mimic the teacher's answer.

Reply:
Thanks, that's very helpful. Already added them into my summary